dependencyManagement/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val TEST_SNAPSHOTS = rootProject.findProperty("testUpstreamSnapshots") == "true"
 
 // This is the version of the upstream instrumentation BOM
-val otelVersion = "1.21.0"
+val otelVersion = "1.21.0-adot1"
 val otelSnapshotVersion = "1.22.0"
```

Comment:
```suggestion
val otelSnapshotVersion = "1.22.0-adot1"
```

Is this needed?

Reply:
No, this is only used in a development workflow when you want to use an upstream snapshot version.

https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/dependencyManagement/build.gradle.kts#L78